examples/tests/valid/util.env.w

```diff
@@ -0,0 +1,26 @@
+bring cloud;
+bring util;
+
+let random_name = "RANDOM123412121212kjhkjskdjkj";
+
+assert(util.Util.env("PATH").length > 0);
+
+let var failed = false;
+try {
+  util.Util.env(random_name);
```

Comment:
Will this in the future be an `unphased` function? (naming tbd...)

Reply:
Yes